tests/golden/test_runner.py

```
@@ -238,5 +238,203 @@ def test_partial_cache_still_fails(self, three_kinds_specs, tmp_path):
+    """Neither ``golden_fn`` nor ``golden_data`` — validation is skipped."""
+
+    def test_skip_validation_passes_even_on_nonsense_outputs(
+        self, three_kinds_specs, tmp_path,
+    ):
+        compiled_dir = tmp_path / "build"
+        compiled_dir.mkdir()
+
+        def fake_execute(work_dir, tensors, **_kwargs):
+            tensors[1][:] = torch.full_like(tensors[1], 9999.0)
+
+        compile_p, exec_p = _patch_compile_and_execute(compiled_dir)
```

Ruff (0.15.10)

[warning] Unpacked variable `compile_p` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)